Sources/SwiftSyntax/Documentation.docc/Contributing/ChangingSwiftSyntax.md

```diff
@@ -22,7 +22,7 @@ swift run --package-path CodeGeneration generate-swiftsyntax
 ## Adding and Removing Syntax Nodes
 
 The files containing the definition of all of the syntax nodes are available
-under the [SyntaxSupport](SyntaxSupport) directory. These files
+under the [SyntaxSupport][SyntaxSupport] directory. These files
```

Comment:
I'm pretty sure the format for links is`[]()`

Reply:
Or that it was GitHub insets when tapping the the button

Reply:
@kimdv, regular Markdown links are indeed `[label](url)`. `[label][link]` is another notation for _reference_ links, that is also valid, as long as in the end of your document, you have a block that describes where each link should actually link to. I.e.

```markdown
this is a valid link! [link label][link target]

[link target]: https://github.com
```

https://daringfireball.net/projects/markdown/syntax#link